colossalai/zero/low_level/low_level_strategy.py

```diff
@@ -306,7 +306,7 @@ def update_master_param(self, master_param):
         padding_size = self.get_param_padding_size(working_param)
         if padding_size > 0:
             working_param = torch.nn.functional.pad(working_param, [0, padding_size])
-        master_param.copy_(working_param.chunk(self._world_size)[self._local_rank])
+        master_param.copy_(working_param.flatten().chunk(self._world_size)[self._local_rank])
```

Comment:
Previously this was [here](https://github.com/hpcaitech/ColossalAI/blob/2ddf624a8631bdb70e9dc7be66059d377c82924d/colossalai/zero/low_level/low_level_optim.py#L990), is it causing any issues?

Reply:
`working_param = p.data.view(-1)` was omitted